src/main/java/seedu/duke/command/HelpCommand.java

```diff
@@ -20,16 +20,15 @@
 
     @Override
     public String execute() {
-
         return HELP_STRING + InterfaceManager.LS + COLOR_WHITE_STRING
                 + AddNoteCommand.getCommandUsageNote() + InterfaceManager.LS + COLOR_CYAN_STRING
                 + AddEventCommand.getCommandUsage() + InterfaceManager.LS + COLOR_WHITE_STRING
                 + CreateTagCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
-                + DeleteNoteCommand.getCommandUsageNote() + InterfaceManager.LS + COLOR_WHITE_STRING
+                + DeleteNoteCommand.getCommandUsage() + InterfaceManager.LS + COLOR_WHITE_STRING
                 + DeleteEventCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
                 + DeleteTagCommand.getCommandUsage() + InterfaceManager.LS + COLOR_WHITE_STRING
-                + EditCommand.getCommandUsageNote() + InterfaceManager.LS + COLOR_CYAN_STRING
-                + EditCommand.getCommandUsageEvent() + InterfaceManager.LS + COLOR_WHITE_STRING
+                + EditNoteCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
+                + EditEventCommand.getCommandUsage() + InterfaceManager.LS + COLOR_WHITE_STRING
                 + ExitCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
                 + FindCommand.getCommandUsage() + InterfaceManager.LS + COLOR_WHITE_STRING
                 + ListEventCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
```

Comment:
Is it perhaps better in the long run if we do it via a ArrayList<Command>? 

```
ArrayList<String> commandHelpList = new ArrayList<String>();
commandHelpList.add(AddNoteCommand.getUsageNote());
commandHelpList.add(AddEventCommand.getUsage());
commandHelpList.add(CreateTagCommand.getUsage());
commandHelpList.add(DeleteNoteCommand.getUsage());
commandHelpList.add(DeleteEventCommand.getUsage());
commandHelpList.add(DeleteTagCommand.getUsage());
commandHelpList.add(EditNoteCommand.getUsage());
commandHelpList.add(EditEventCommand.getUsage());
commandHelpList.add(ExitCommand.getUsage());
commandHelpList.add(FindCommand.getUsage());
commandHelpList.add(ListEventCommand.getUsage());

int i = 0;
resultString = "";
for (String command : commandHelpList) {
    String color = (i % 2 == 0) ? COLOR_CYAN_STRING : COLOR_WHITE_STRING;
    i++;
    resultString += command + InterfaceManager.LS + color;
}
return resultString;
```

Reply:
Don't really think there is a need for a function because this help string will be a constant. Can actually merge with the declaration of the HELP_STRING. @R-Ramana your view?

Reply:
I'm just afraid if we add more commands down the line we need human intervention to always check if the colour ordering is correct